app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from typing import List
+
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(players: List[Player]) -> int:
+    sum_rating = 0
+    for player in players:
+        sum_rating += player.get_rating()
+    return sum_rating
+
+
+def elves_concert(elves: list[Elf]) -> None:
```

Comment:
The type hint `list[Elf]` should be replaced with `List[Elf]` to be consistent with the imported `List` from the `typing` module.